labman/gui/static/js/plateViewer.js

```diff
@@ -129,6 +129,13 @@ PlateViewer.prototype.initialize = function (rows, cols) {
                        cssClass: 'slick-header-column',
                        headerCssClass: 'full-height-header'}];
 
+  // Fetch the blank names before initializing the grid. This way we only make
+  // one request to get this data instead of one per cell instantiation.
+  var blanksRequest = $.ajax({
+    dataType: 'json',
+    url: '/sample/control?term=',
```

Comment:
To make sure I understand this, by not requesting a term or set of terms, we get back all of the possible controls?

Reply:
Okay, thanks. Would it be possible to add a comment indicating as such as the behavior is implicit?

Reply:
Will do.